ddtrace/contrib/django/middleware.py

```diff
@@ -56,6 +56,8 @@ def process_response(self, request, response):
             span = _get_req_span(request)
             if span:
                 span.set_tag(http.STATUS_CODE, response.status_code)
+                if response.status_code == 500:  # Some exceptions are not handled by `process_exception`
+                    span.set_traceback()
```

Comment:
Ok, I'have done an example where I couldn't catch the 500 status code with process_exception, but I'll check the doc to see if there is an other way to catch this exception. +1 for the unit test